Actions/ReadSecrets/ReadSecrets.ps1

```diff
@@ -77,6 +77,21 @@ try {
                 }
             }
         }
+        # Look through installApps and installTestApps for secrets and add them to the collection of secrets to get
+        foreach($installSettingsKey in @('installApps','installTestApps')) {
+            if ($settings.Keys -contains $installSettingsKey) {
+                $settings."$installSettingsKey" | ForEach-Object {
+                    # If any of the installApps URLs contains '${{SECRETNAME}}' we need to get the secret
+                    $pattern = '.*(\$\{\{\s*([^}]+?)\s*\}\}).*'
```

Comment:
Minor: not sure why you match two groups here. `'.*\$\{\{(.*)\}\}.*'` should suffice.

Also, could there be a case that, for whatever reason, two secrets are used in one entry?
Not saying AL-Go should support this, just making an observation.

Reply:
Got it why you also match the `${{ ... }}` group: to re-use the same pattern when replacing with the actual secret.

Reply:
Correct - I could remove one parentheses here, but for the sake of using the same and not having to decipher.